schemas/data/poi-entries.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/metrics/placecontext/poi-entries",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "poi-entries",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/metric"],
+  "description":
+    "The place context metric poi-entries describes the number of entries into a Point of Interest (POI).",
+  "definitions": {
+    "metric": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "const": "https://ns.adobe.com/xdm/data/metrics/placecontext/poi-entries",
+          "description": "The unique identifier of this metric."
+        },
+        "schema:name": {
+          "type": "string",
+          "const": "placecontext: poi-entries",
+          "description":
+            "The human-readable name of the metric. The name can be used in user interfaces and does not have to be unique."
+        },
+        "xdm:measurement": {
```

Comment:
I don't understand this one...

Reply:
Metrics are a definition of a measure. A measure can be anything that is being counted or added together. In this case the value in the event is always 1, because there is one entry. If the measure were a currency it would be a summing measure.

Reply:
So are we saying that you could be a 1 or a 0. We are in or you are out? If so, this doesn't map with POI interaction which is enter or exit. 

Do we have two different things? One is an event which is POI Interaction. The other is contextual data attached to another type of event say purchase. I can purchase at home on my mobile app. No defined POI would be associated. I can purchase in store from the Point of sale there. Point of interest associated. I could purchase in the store from my app. POI associated to that online purchase event. 

Am I following this right?

Reply:
Yes you follow correctly.